templates/kubernetes/terraform/environments/stage/main.tf

```diff
@@ -65,6 +65,9 @@ module "kubernetes" {
     # Add additional mappings here
   ]
 
+  # Version of default database user password for application to access database - change the version if you want to renew the password
+  db_app_password_version = "v1.0"
```

Comment:
How about just make this the default value and then don't specify it here, and then make sure this comment is in the variables file. This setting is unlikely to change often, it would be nice to keep this file as slim as possible.

Reply:
sure, good!